target/linux/ipq806x/base-files/lib/upgrade/platform.sh

```diff
@@ -10,6 +10,10 @@ platform_check_image() {
 
 platform_do_upgrade() {
 	case "$(board_name)" in
+	askey,rt4230w-rev6)
+		PART_NAME="ubi"
```

Comment:
PART_NAME is for default_do_upgrade. nand_do_upgrade uses CI_UBIPART, and the default value is ubi. So probably you can remove this line.

Reply:
@lmore377 Why did you mark this as resolved?

Reply:
I removed it and sysupgrade still works.